src/Files.App/Views/PaneHolderPage.xaml.cs

```diff
@@ -342,6 +346,8 @@ private void Pane_Loaded(object sender, RoutedEventArgs e)
 		{
 			((UIElement)sender).GotFocus += Pane_GotFocus;
 			((UIElement)sender).RightTapped += Pane_RightTapped;
+
+			PaneLeft.RootGrid.Translation = new System.Numerics.Vector3(0, 0, 8);
```

Comment:
I moved this to the loaded event, otherwise it would have triggered whenever the user navigated to a different page.